pom.xml

```diff
@@ -164,6 +165,90 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!--
+      The profile deploys all artifacts to Github Packages repository (if you
+      define github.repo property). It also skips default deployment plugin.
+      @see http://www.jcabi.com/jcabi-parent/profile-github.html
+      -->
+      <id>github-packages</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <artifactId>maven-deploy-plugin</artifactId>
+              <configuration>
+                <skip>true</skip>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>validate-github-repo</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+                <phase>validate</phase>
+                <configuration>
+                  <rules>
+                    <requireProperty>
+                      <property>github.repo</property>
+                      <message>You must define 'github.repo' property to deploy to Github Packages.</message>
+                      <regex>^(?!\s*$).+</regex>
+                    </requireProperty>
+                  </rules>
+                  <fail>true</fail>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>maven-source-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>jcabi-sources</id>
+                <configuration>
+                  <forceCreation>true</forceCreation>
+                </configuration>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <!--
+            Generate -javadoc.jar artifact.
+            -->
+            <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
@fabriciofx we already have this confirmation in the POM. Maybe no need to duplicate it here?

Reply:
@yegor256 done.